lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -123,7 +121,7 @@ public Translation2d FieldRelativeTranslation2d() {
      *         for translation and radians for rotation
      */
     public Transform2d robotRelativeTransform2d() {
-        return new Transform2d(getX(), getY(), robotRelativeAngleToNote());
+        return new Transform2d(RobotRelativeTranslation2d(), robotRelativeAngleToNote());
```

Comment:
this transform involves a translation to the note and also a rotation to face it along the line from the robot position to the note position.  since these two things, the translation and the rotation, actually describe the same thing, just let the caller decide how they want to rotate.